src/planscape/planscape/tests/factories.py

```diff
@@ -0,0 +1,23 @@
+from datetime import datetime
+
+import factory
+import factory.fuzzy
+import pytz
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = User
+
+    username = factory.Faker("user_name")
```

Comment:
Is there any need to guard against the (probably very unlikely) event of collisions on user_name, or is this just a very remote risk where the worst-case is that the DB rejects a username during a test?
I see you have a Sequence for scenario names above, so was  wondering if we need that here.